pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -348,6 +348,11 @@ func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kube
 	if err != nil {
 		return err
 	}
+
+	// This also add the current pod to the running pods list because for rkt, when we set up the network,
+	// the pod is not currently running.
+	runningPods = append(runningPods, &hostport.RunningPod{Pod: pod, IP: ip4})
```

Comment:
getRunningPods is retrieving running pods with an ip in podIPs map. The current pod should have it.  Unless GetPods in rkt does not return the current pod.